kf4cic.c

```diff
@@ -280,6 +305,11 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *_table)
 	ret = load_and_start_tos(image);
 	if (EFI_ERROR(ret))
 		return ret;
+
+	debug(L"Create VT-d activate event...\n");
+	ret = create_vtd_activate_event();
```

Comment:
should it be called before tos launching (line 305)?

Reply:
I think if start trusty failed, then no need to create such event?